### ISSUE TITLE
How to compute the gradient of reward w.r.t. torso mass?

### ISSUE THREAD

**Issue by ZJP-c4cld** [openai/gym#2812]
I want to construct a neural network in pytorch which outputs appropriate values of mujoco model parameters. But this requires that mujoco model parameters are converted into pytorch tensors , and mujoco environments run well with these converted model parameters, and compute the gradient of reward w.r.t the mujoco model parameters.
Anyone know how to achieve this goal?

**Comment by pseudo-rnd-thoughts** [comment]
I don't understand how this is an issue for gym, this seems like a research question so I would talk to academics about it as they will have better understanding of research on this topic.
If you can find any issue with gym that prevents you to do something then please reopen the issue

**Comment by kevinzakka** [comment]
@ZJP-c4cld Would be great if you didn't open the same issue across 4+ repositories, not great etiquette :)
